src/lib.rs

```diff
@@ -43,6 +43,7 @@ pub use index::*;
 pub use manifest::{find, get_dep_version, set_dep_version, LocalManifest, Manifest};
 pub use metadata::manifest_from_pkgid;
 pub use registry::registry_url;
+pub use toml_edit;
```

Comment:
As this crate is **not** meant to be used as a library (see semver compatibility](https://docs.rs/cargo-edit/latest/cargo_edit/#semver-compatibility), I would prefer this to be reverted

Reply:
Fair enough, I've dropped commit acb2f2bf1403932ff49b9e7b280b62421570ab7c.